iis-wf/iis-wf-primary/src/main/resources/eu/dnetlib/iis/wf/primary/processing/oozie_app/workflow.xml

```diff
@@ -2015,7 +2015,7 @@
             <arg>-Preport.processing.documentSimilarity.duration=transformers_documentssimilarity,documentssimilarity_chain</arg>
             <arg>-Preport.processing.citationMatching.direct.duration=citationmatching_direct</arg>
             <arg>-Preport.processing.citationMatching.fuzzy.duration=citationmatching_fuzzy</arg>
-            <arg>-Preport.processing.citationMatching.total.duration=citationmatching_fuzzy,citationmatching_direct,transformers_citations_from_referencemetadata,citations_union</arg>
```

Comment:
I think in this particular case we should not rename `total` because it refers to total amount of time required by the all citation matching phases (not to the total number of references which are deduped). We should probably drop the `total` part of the key to stay in compliance with other algos (`processing.documentSimilarity.duration`, `processing.affiliationMatching.duration`, `processing.documentClassification.duration`).

Reply:
Ok, dropping `total` part, should've spotted this one by looking at primary execution times because the one with `deduped` part stands out :/